validator/src/bootstrap.rs

```diff
@@ -653,16 +653,16 @@ fn get_highest_local_snapshot_hash(
 /// 3. have the highest full snapshot slot of (2)
 fn get_peer_snapshot_hashes(
     cluster_info: &ClusterInfo,
-    validator_config: &ValidatorConfig,
     rpc_peers: &[ContactInfo],
+    known_validators: &Option<HashSet<Pubkey>>,
```

Comment:
nit: Please move the `&`to the type inside the Option:

```suggestion
    known_validators: Option<&HashSet<Pubkey>>,
```

Callers should be able to use `validator_config.known_validators.as_ref()`.

Reply:
I removed `validator_config`. Could you please check last commit one more time? Sorry, I’m from a phone, it’s hard to insert code blocks :(
And it worked, I compiled and ran.

Reply:
@brooksprumo I hope I understood what you suggested correctly cdc649cd228622dc7f7a1350933e126a0cf1dd2d